### PR TITLE
Restrict pointOfContact read/write permissions for consultant manager, financial analyst, marketing and staff member roles

### LINKED ISSUE #2150: Partner > pointOfContact > User can read the "pointOfContact" details even that user has "No read" permission
**Roles:** ConsultantManager

**Describe the bug**
Partner > pointOfContact > User can read the `pointOfContact` details even that user has `No read` permission

**Steps to Reproduce**
1. Login as Admin user
2. Create a partner and go to the partner detail page
3. Login as a consultant manager user
4. Open created a partner

**Expected Result**
Partner details should not be displayed

**Actual Result**
User can read the `pointOfContact` details even the user has `No read` permission
 
See video - 

https://user-images.githubusercontent.com/5212585/133405866-5df25c0e-56cc-448b-a5c1-d54b4f75cd12.mp4

image

**FYI:** @ParkeBrown @michaelmarshall 

cord-field# 851d128
CordAPIV3# 01818ef

### PR DESCRIPTION
**Pull request by daniel-leod**
Fixes #2150

### CODE REVIEW

File: src/components/authorization/roles/marketing-global.role.ts [hunk 1]
```diff
@@ -277,7 +277,7 @@ export const Marketing = new DbRole({
       canList: true,
       properties: [
         { propertyName: 'organization', permission: { read, }, },
-        { propertyName: 'pointOfContact', permission: { read, }, },
+        { propertyName: 'pointOfContact', permission: { }, },
         { propertyName: 'types', permission: { read, }, },
         { propertyName: 'financialReportingTypes', permission: { read, }, },
         { propertyName: 'pmcEntityCode', permission: { read, }, },
```
Comment (bryanjnelson) [inline src/components/authorization/roles/marketing-global.role.ts:283]: Similarly here what you did so far looks good, but probably remove the read permission from `pmcEntityCode` entirely, and also take the read permission for `organization` down to Low Sensitivity to match the Sheet:

Screen Shot 2021-11-05 at 2 07 41 PM

File: src/components/authorization/roles/marketing-on-project.role.ts [hunk 2]
```diff
@@ -277,7 +277,7 @@ export const MarketingOnProject = new DbRole({
       canList: true,
       properties: [
         { propertyName: 'organization', permission: { read, }, },
-        { propertyName: 'pointOfContact', permission: { read, }, },
+        { propertyName: 'pointOfContact', permission: { }, },
         { propertyName: 'types', permission: { read, }, },
         { propertyName: 'financialReportingTypes', permission: { read, }, },
         { propertyName: 'pmcEntityCode', permission: { read, }, },
```
Comment (bryanjnelson) [inline src/components/authorization/roles/marketing-on-project.role.ts:283]: The `pointOfContact` looks good; you can do the same thing for `pmcEntityCode`.

File: src/components/authorization/roles/staff-member.role.ts [hunk 3]
```diff
@@ -274,7 +274,7 @@ export const StaffMember = new DbRole({
       canList: true,
       properties: [
         { propertyName: 'organization', permission: { read, }, },
-        { propertyName: 'pointOfContact', permission: { read, }, },
+        { propertyName: 'pointOfContact', permission: { }, },
         { propertyName: 'types', permission: { read, }, },
         { propertyName: 'financialReportingTypes', permission: { read, }, },
         { propertyName: 'pmcEntityCode', permission: { read, }, },
```
Comment (bryanjnelson) [inline src/components/authorization/roles/staff-member.role.ts:280]: Again `pointOfContact` looks good; you can do the same with `pmcEntityCode`.  And also take `organization` down to Low Sensitivity.

**Review by bryanjnelson — CHANGES_REQUESTED** [review]

**Review by bryanjnelson — CHANGES_REQUESTED** [review]

**Review by bryanjnelson — APPROVED** [review]

**Comment by bryanjnelson** [comment]
@daniel-leod - Looks good!
@CarsonF - Ready for merge!

**Review by CarsonF — APPROVED** [review]

**Pull request merged**